### PR TITLE
Solution article for 2024 Day 01

### PR DESCRIPTION
**Pull request by spamegg1**
- Also added link to my community solution

### CODE REVIEW

File: docs/2024/puzzles/day01.md [hunk 1]
```diff
@@ -6,7 +6,69 @@ import Solver from "../../../../../website/src/components/Solver.js"
...
+
+### Part 1
+
+Now that the lefts and rights are sorted in increasing order, we can zip them,
+so that the first smallest on the left is paired with the first smallest on the right,
+the second smallest on the left is paired with the second smallest on the right, and so on.
+Then we can find the distances between them, and sum the distances:
+
+```scala
+def part1(input: String): String =
```
Comment (bishabosha) [inline docs/2024/puzzles/day01.md:43]:
```suggestion
def part1(input: String): Int =
```

File: docs/2024/puzzles/day01.md [hunk 2]
```diff
@@ -6,7 +6,69 @@ import Solver from "../../../../../website/src/components/Solver.js"
...
+
+### Part 2
+
+Very similar, but instead of distances, we find a left number's similarity on the right list.
+We do this by counting how many times the left number occurs on the right list,
+then multiply that count by the number itself.
+Finally we sum the similarity scores of all the left numbers:
+
+```scala
+def part2(input: String): String =
```
Comment (bishabosha) [inline docs/2024/puzzles/day01.md:60]:
```suggestion
def part2(input: String): Int =
```

File: docs/2024/puzzles/day01.md
Comment (KacperFKorban) [inline docs/2024/puzzles/day01.md:1]: Could you add yourself as the author at the beginning of the post, just like with the previous ones?
e.g. https://scalacenter.github.io/scala-advent-of-code/2023/puzzles/day01/

**Review by KacperFKorban — APPROVED** [review]
Looks good, could you resolve the conflicts, so we can merge?
BTW, I created a schedule for writing the articles and I've added you for the first two days since you already submitted them :smile:
You can register for some more days if you want.

**Comment by SethTisue** [comment]
@KacperFKorban I had told @adpi2 I would do the scheduling, and I was going to start on it today (after a long weekend of being sick and then American holiday), but it seems that maybe you and @gkepka are on it and I'm not needed...?

**Comment by SethTisue** [comment]
thank you @gkepka for helping out 👍

**Pull request merged**